backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -202,3 +210,19 @@ internal fun formatGenerics(buffer: Appendable, generics:List<String>) {
     }
 }
 
+private fun DeclarationDescriptor.extractKDocString(): String? {
+    if (this is DeclarationDescriptorWithSource) {
+        val psi = source.getPsi()
+        if (psi is KtDeclaration) {
+            if (psi is KtPrimaryConstructor)
+                return null  // to be rendered with class itself
+            val kdoc = psi.docComment
+            if (kdoc != null) {
+                return kdoc.getDefaultSection().parent.text
+            }
+        }
+    }
```

Comment:
The part above is quite similar to [`fun DeclarationDescriptor.findKDoc(): KDocTag?` in kotlin repo](https://github.com/JetBrains/kotlin/pull/3978/files#diff-27ccd26943ccb07ece6106b91696eee8cf9e627c6c531f2904fe6cf4ca0f2323R90).
Also the behaviour should match. So it makes some sense to keep only one of these functions.

Reply:
Fixed.